README.md

```diff
@@ -20,22 +20,22 @@ This script is useful for performing image analysis on locally built container i
 
 Sysdig Inline Analyzer --
 
-  Script for performing analysis on local docker images, utilizing the Sysdig analyzer subsystem.
+  Script for performing analysis on local container images, utilizing the Sysdig analyzer subsystem.
   After image is analyzed, the resulting image archive is sent to a remote Sysdig installation
   using the -s <URL> option. This allows inline analysis data to be persisted & utilized for reporting.
 
   Images should be built & tagged locally.
 
     Usage: inline_scan.sh analyze -s <SYSDIG_REMOTE_URL> -k <API Token> [ OPTIONS ] <FULL_IMAGE_TAG>
 
-      -s <TEXT>  [required] URL to Sysdig Secure URL (ex: -s 'https://secure-sysdig.com')
+      -s <TEXT>  [required] Sysdig Secure URL (ex: -s 'https://secure-sysdig.svc.cluster.local')
```

Comment:
Everything else looks good and makes sense. 
Nit: But I am wondering shouldn't our uri be `https://secure.sysdig.com` consistently everywhere ? I am not sure I understand why it needs to be instead `https://secure-sysdig.svc.cluster.local`? Let me know what context I am missing.
cc: @yathi-sysdig